src/main/java/DBot.java

```diff
@@ -0,0 +1,111 @@
+import java.util.*;
+
+public class DBot {
+    private static boolean isOn;
+    private static List<Task> datas;
+
+    public static void main(String[] args) {
+        isOn = true;
+        datas = new ArrayList<>();
+
+        String greeting = "____________________________________________________________\n" +
+                "Hello! I'm DBot\nWhat can I do for you?\n"
+                + "____________________________________________________________\n";
+        System.out.println(greeting);
+
+        Scanner in = new Scanner(System.in);
+
+        while (isOn) {
+            System.out.print("Command: ");
+            String line = in.nextLine().strip();
+            System.out.println("____________________________________________________________");
+
+            if (line.equals("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                isOn = false;
+            } else if (line.equals("list")) {
+                list();
+            } else if (line.startsWith("mark ")) {
+                mark(line);
+            } else if (line.startsWith("unmark ")) {
+                unmark(line);
+            } else if (line.startsWith("todo ")) {
+                todo(line);
+            } else if (line.startsWith("event ")) {
+                event(line);
+            } else if (line.startsWith("deadline ")) {
+                deadline(line);
+            } else {
+                add(line);
+            }
```

Comment:
Perhaps you could refactor this into another function? This chunk is long and hard to read.